src/pods/edit-realtion/edit-relation.business.ts

```diff
@@ -37,11 +37,16 @@ export const mapRelationsTipeToDropdonwVm = (): DropdownOptionVm[] => [
 // #118 Add unit tests to edit relation business
```

Comment:
typo

mapRelationTipe

should be Type

I have installed the codespell checker extension (english) and (spanish) it marks in blue where there is some typo

Reply:
It's fixed